src/OpenFeature.Contrib.Providers.Flagd/FlagdProvider.cs

```diff
@@ -0,0 +1,387 @@
+        /// </summary>
+        /// <param name="src">The struct</param>
+        /// <returns>A Value object representing the given struct</returns>
+        private static Value ConvertObjectToValue(Struct src) =>
+            new Value(new Structure(src.Fields
+                .ToDictionary(entry => entry.Key, entry => ConvertToValue(entry.Value))));
+
+        /// <summary>   
+        ///     ConvertToValue converts the given ProtoValue to a Value.
+        /// </summary>
+        /// <param name="src">The value, represented as ProtoValue</param>
+        /// <returns>A Value object representing the given value</returns>
+        private static Value ConvertToValue(ProtoValue src)
+        {
+            switch (src.KindCase)
+            {
+                case ProtoValue.KindOneofCase.ListValue:
+                    return new Value(src.ListValue.Values.Select(ConvertToValue).ToList());
+                case ProtoValue.KindOneofCase.StructValue:
+                    return new Value(ConvertObjectToValue(src.StructValue));
+                case ProtoValue.KindOneofCase.None:
+                case ProtoValue.KindOneofCase.NullValue:
+                case ProtoValue.KindOneofCase.NumberValue:
+                case ProtoValue.KindOneofCase.StringValue:
+                case ProtoValue.KindOneofCase.BoolValue:
+                default:
+                    return ConvertToPrimitiveValue(src);
+            }
+        }
+
+        /// <summary>   
+        ///     ConvertToPrimitiveValue converts the given ProtoValue to a Value.
+        /// </summary>
+        /// <param name="value">The value, represented as ProtoValue</param>
+        /// <returns>A Value object representing the given value as a primitive data type</returns>
+        private static Value ConvertToPrimitiveValue(ProtoValue value)
+        {
+            switch (value.KindCase)
+            {
+                case ProtoValue.KindOneofCase.BoolValue:
+                    return new Value(value.BoolValue);
+                case ProtoValue.KindOneofCase.StringValue:
+                    return new Value(value.StringValue);
+                case ProtoValue.KindOneofCase.NumberValue:
+                    return new Value(value.NumberValue);
+                case ProtoValue.KindOneofCase.NullValue:
+                case ProtoValue.KindOneofCase.StructValue:
+                case ProtoValue.KindOneofCase.ListValue:
+                case ProtoValue.KindOneofCase.None:
+                default:
+                    return new Value();
+            }
+        }
+
+        private static Service.ServiceClient buildClientForPlatform(Uri url)
+        {
+#if NETSTANDARD2_0
+            return new Service.ServiceClient(GrpcChannel.ForAddress(url));
+#else
+            return new Service.ServiceClient(GrpcChannel.ForAddress(url, new GrpcChannelOptions
+            {
+                HttpHandler = new WinHttpHandler()
+            }));
+#endif
```

Comment:
@benjiro does this look right to you? It seems to pass in all the test scenarios not (462 was previously failing).